lib/aiken-design-patterns/linked-list/ordered.ak

```diff
@@ -0,0 +1,251 @@
+use aiken/collection/dict.{Dict}
+use aiken/collection/list
+use aiken/primitive/bytearray
+use aiken_design_patterns/linked_list/types.{Empty, Key, NodeDatum, NodeKey}
+use aiken_design_patterns/linked_list/unordered
+use aiken_design_patterns/linked_list/utils.{
+  get_key, get_link, get_raw_key_and_link,
+}
+use cardano/address.{Address}
+use cardano/assets.{AssetName, flatten}
+use cardano/transaction.{InlineDatum, Input, Output}
+
+pub fn list_state_transition(node_mint: Dict<AssetName, Int>) {
+  unordered.list_state_transition(node_mint)
+}
+
+pub fn modify_data(
+  node_mint: Dict<AssetName, Int>,
+  own_input: Input,
```

Comment:
We don't use the input id (i.e. output reference) part of the input, so let's just make the argument an Output.
```suggestion
  own_input: Output,
```

Reply:
Resolved by https://github.com/Anastasia-Labs/aiken-design-patterns/pull/12